src/Sonarr.Api.V3/Episodes/RenameEpisodeController.cs

```diff
@@ -26,5 +27,17 @@ public List<RenameEpisodeResource> GetEpisodes(int seriesId, int? seasonNumber)
 
             return _renameEpisodeFileService.GetRenamePreviews(seriesId).ToResource();
         }
+
+        [HttpGet("multiple")]
+        [Produces("application/json")]
+        public List<RenameEpisodeResource> GetEpisodes(List<int> seriesIds)
```

Comment:
I was getting a 415 error without the `[FromQuery]]`

```suggestion
        public List<RenameEpisodeResource> GetEpisodes([FromQuery] List<int> seriesIds)
```